src/Authentication/UserAuth.cs

```diff
@@ -34,43 +35,59 @@ public static bool Registration(string email, string pass)
             db.Add(user);
             db.SaveChanges();
 
-            //Handler.UserId = user.Id;
-            return true;
+            return user.Id;
         }
 
-        public static bool Login(string email, string pass)
+        public static int Login(string email, string pass)
         {
             using var db = new DatabaseContext();
-            var userInfo = db.Users.FirstOrDefault(a => a.Email == email); //Find user and pass in db and check if matches
+            var user = db.Users.FirstOrDefault(a => a.Email == email); //Find user and pass in db and check if matches
 
-            if (userInfo == null)
+            if (user == null)
             {
-                return false;
+                //User couldn't be found
+                return -1;
             }
 
-            if (!HashingProcessor.AreEqual(pass, userInfo.Password, userInfo.Salt))
+            if (!HashingProcessor.AreEqual(pass, user.Password, user.Salt))
             {
-                return false;
+                //Password is wrong
+                return -2;
             }
-            //Handler.UserId = userInfo.Id;
-            return true;
+            return user.Id;
+        }
+
+        public static int GetUserIdByEmail(string email)
+        {
+            if (email is null)
+            {
+                return -1;
+            }
```

Comment:
Nice optimisation, I thought why would we need this since if it is empty it won't be found in database, and in this case it wont even make connection to the DB.